### ISSUE TITLE
MaterialDatePicker get selection on null object reference

### ISSUE THREAD

**Issue by DanilaRazvan** [material-components/material-components-android#1968]
Description:
After building the date picker with an initial selection, if I want to retrieve that selection this exception is thrown:
```
java.lang.NullPointerException: Attempt to invoke interface method 'java.lang.Object com.google.android.material.datepicker.DateSelector.getSelection()' on a null object reference
```
Expected behavior:
If an initial selection is set in the builder, retrieving that selection could be possible

Source code:
```
    val datePickerBuilder = MaterialDatePicker.Builder.dateRangePicker()

    datePickerBuilder.apply {
      setTitleText("SELECT A DATE")
      setTheme(R.style.MaterialCalendarTheme)
      setSelection(
        Pair(
          MaterialDatePicker.todayInUtcMilliseconds(),
          MaterialDatePicker.todayInUtcMilliseconds()
        )
      )
    }

    val dp = datePickerBuilder.build()
    dp.selection // throws exception
```

Material Library version: 1.3.0-beta01

**Comment by ikim24** [comment]
Some context: things like the `DateSelector` (and therefore the selection), that are set from the builder are currently not available on `MaterialDatePicker` until the fragment has been created (specifically it gets set in `onCreate`).

**Comment by drchen** [comment]
It's been fixed in 1.5.0 alpha.
